src/Trinity.C/src/Trinity/Configuration/TrinityConfig.cpp

```diff
@@ -181,29 +160,13 @@ namespace TrinityConfig
     }
 
     /// <summary>
-    /// Value = 2G + MemoryReserveUnit * 32
+    /// Total reserved space in bytes for a trunk =
+    /// TrunkLength (2GB) + ReserveEntriesPerMTHash * (CellEntry+MTEntry) + BucketCount * (Bucket+Lock)
+    /// Rounded up to page boundary.
     /// </summary>
     uint64_t ReservedSpacePerTrunk()
     {
-        switch (CapacityProfile)
-        {
-        case StorageCapacityProfile::Max32G:
-            return 0x180000000; // 6G
-        case StorageCapacityProfile::Max16G:
-            return 0x100000000; // 4G
-        case StorageCapacityProfile::Max4G:
-            return 0xA0000000; // 2.5 G
-        case StorageCapacityProfile::Max2G:
-            return 0x90000000; // 2 G + 256 M
-        case StorageCapacityProfile::Max1G:
-            return 0x88000000; // 2 G + 128 M
-        case StorageCapacityProfile::Max512M:
-            return 0x84000000; // 2 G + 64 M
-        case StorageCapacityProfile::Max256M:
-            return 0x82000000; // 2 G + 32 M
-        default:
-            return 0xC0000000;// 3G
-        }
+        return Memory::RoundUpToPage_64(TrinityConfig::TwoGigabytes + Storage::MTHash::MTHashReservedSpace());
```

Comment:
Different values from original ones. Will cause any issues?

Reply:
The original value is wrong.